nevergrad/benchmark/ioh.md

```diff
@@ -0,0 +1,7 @@
+# Exporting data to IOH Profiler.
```

Comment:
There should not be md files within the code. You could probably simply add a section in `docs/benchmarking.md` instead, it will have more visibility as well.